resources/assets/less/bem/contest.less

```diff
@@ -47,6 +48,55 @@
     }
   }
 
+  &__accordian {
+    background: #1A2223;
+    padding: @spacing * 2;
+  }
+
+  &__group {
+    margin: @spacing auto;
+    max-width: 600px;
+    background: #161a1b;
+    border: none;
+    .default-box-shadow();
+  }
+
+  &__group-heading {
+    display: flex;
+    justify-content: space-between;
+    padding: @spacing;
+    background: #161a1b;
+    border-top-left-radius: 5px;
+    border-top-right-radius: 5px;
+
+    .white-link();
+    &:hover, &:visited, &:link, &:active {
+      .white-link();
```

Comment:
the generated css seems... wrong. Probably just set the color directly or fix the `white-link` function.

![](https://puu.sh/sl9ty/763a9fac9d.png)